### DISCUSSION TITLE
Authentication failed for 'https://github.com/maazg10/gdec.git/'

### DISCUSSION THREAD

**Discussion by maazg10** [community/community#52491] [New to GitHub]
Hello Everyone i hope that you are doing well Today is my first day to join Github community i have question related to **Authentication failed for '**    i request you  to please help me how to solve this issue

**Comment by neutrinomuon** [comment]
Hi, @maazg10,

The error message "Authentication failed for 'https://github.com/maazg10/gdec.git/'" suggests that there was an issue with the authentication credentials provided while trying to access the Git repository. This can happen if the username and password or access token used for authentication is incorrect or if the user does not have the necessary permissions to access the repository. Can you provide further details? How are you trying to access it? For instance, If you are using regular `git pull` or `git push` commands, you need a token (**not your GitHub password**), for that.

Regarding the repository, I've checked and it seems empty, is that right? 

Please, let me know if it helps.

**Reply by maazg10** [reply]
thank you so much for your feedback yes my repository empty I'm  starting new learning so this is why a lot of problems when i use git pull and git push then   show enter username and password when i enter password then error appear i use Linux operating system it is possible to guide me

**Reply by airtower-luna** [reply]
As @neutrinomuon mentioned, you cannot use your password for `git push` or other Git operations (but you still need it to log in). You need to [create a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) and enter that instead when Git asks for your password. When creating the token make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Reply by maazg10** [reply]
thank you so much ok i try
